### ISSUE TITLE
Things to do.

### ISSUE THREAD

**Issue by ZTCooper** [Debug-ging/fwwbDataAnalysis#1]
- [x] Use **pandas.get_dummies** to one-hot encode the data.

- [x] Preliminarily fit data to the linear regression model.

- [ ] Gotta read the [sklearn document](http://scikit-learn.org/stable/supervised_learning.html#supervised-learning).

- [x] Separate the data into **training set** and **test set**( maybe add **cross validation set**.

- [ ] Figure out how to convert input features into one-hot.

- [ ]
